x-pack/plugins/timelines/common/api/search_strategy/model/filter_query.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Going through some of these schema's, I feel like these should be available at the Kibana level, because I can see it being replicated across plugins. Not something for you to do, but worth adding a ticket to circle back and see if Kibana platform can package these